test/std-test.cc

```diff
@@ -16,7 +16,11 @@ TEST(std_test, path) {
             "\"foo\\\"bar.txt\"");
 
 #  ifdef _WIN32
-  EXPECT_EQ(fmt::format("{}", std::filesystem::path(L"Файл.txt")),
+  // File.txt in Russian.
+  // Do not use utf-8 encoded chars in this file.
+  const wchar_t cyrillic_path[] = {0x424, 0x430, 0x439, 0x43b, 0x2e,
```

Comment:
nit: cyrillic_path -> unicode_path because the test is for unicode, not just cyrillic text.

Reply:
Unexpected Linux CI error:
```
E: Failed to fetch http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu/pool/main/g/gcc-11/libgomp1_11.1.0-1ubuntu1~20.04_amd64.deb  Cannot initiate the connection to ppa.launchpad.net:80 (2620:2d:4000:1::3e). - connect (101: Network is unreachable) Could not connect to ppa.launchpad.net:80 (185.125.190.52), connection timed out [IP: 185.125.190.52 80]
```